applications/ParticleMechanicsApplication/python_scripts/python_solvers_wrapper_particle.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+
+
+def CreateSolver(model, custom_settings):
+
+    if (type(model) != KratosMultiphysics.Model):
+        raise Exception("input is expected to be provided as a Kratos Model object")
+
+    if (type(custom_settings) != KratosMultiphysics.Parameters):
+        raise Exception("input is expected to be provided as a Kratos Parameters object")
+
+    parallelism = custom_settings["problem_data"]["parallel_type"].GetString()
+    solver_type = custom_settings["solver_settings"]["solver_type"].GetString()
+
+    # Solvers for OpenMP parallelism
+    if (parallelism == "OpenMP"):
+        if (solver_type == "Static" or solver_type == "Static" or
```

Comment:
according to the style-guide those should be lower-case => it would be good to add them as option